src/jsx/Timeline.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import Post from "./Post.jsx";
+
+class Timeline extends Component {
+
+  	constructor() {
+    	super();
+  	}
+
+  	generatePosts() {
+  		return this.props.timeline.map((post, index) => {
+        let options = { month: 'short', day: 'numeric' };
+        let createdAt  = new Date(post.createdAt).toLocaleDateString("en-US", options);
+        return <Post key={index}
```

Comment:
you could move the date conversion to a lower level

Reply:
at first when i moved my date conversion up a level to timeline, i was thinking it'd be reasonable for post to render whatever text/date is passed in, but yes it also makes sense for posts to always have the same date format